### ISSUE TITLE
Faraday Server - AttributeError: 'module' object has no attribute 'in_'

### ISSUE THREAD

**Issue by daniejstriata** [infobyte/faraday#283]
## Faraday version
Faraday v3.1.1
## Component Name
Server
## Steps to reproduce
I originally installed Faraday from the version 3.0's zip archive. I cloned from git the new 3.1.1 release into another directory and replaced the original working install with the contents from the git directory. I installed the requirements and ran the server. Running the server it gives me the error above:

>  python2 faraday-server.py --start
> Traceback (most recent call last):
>   File "faraday-server.py", line 18, in <module>
>     from server.models import db, Workspace, User
>   File "/home/danie.dejager/Tools/faraday-3.0/server/models.py", line 1768, in <module>
>     import server.events
>   File "/home/danie.dejager/Tools/faraday-3.0/server/events.py", line 20, in <module>
>     from server.websocket_factories import changes_queue
>   File "/home/danie.dejager/Tools/faraday-3.0/server/websocket_factories.py", line 17, in <module>
>     from twisted.internet import reactor
>   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/reactor.py", line 38, in <module>
>     from twisted.internet import default
>   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/default.py", line 56, in <module>
>     install = _getInstallFunction(platform)
>   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/default.py", line 44, in _getInstallFunction
>     from twisted.internet.epollreactor import install
>   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/epollreactor.py", line 24, in <module>
>     from twisted.internet import posixbase
>   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/posixbase.py", line 18, in <module>
>     from twisted.internet import error, udp, tcp
>   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/udp.py", line 53, in <module>
>     from twisted.internet import base, defer, address
>   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/base.py", line 27, in <module>
>     from twisted.internet._resolver import (
>   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/_resolver.py", line 25, in <module>
>     from twisted.internet.address import IPv4Address, IPv6Address
>   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/address.py", line 23, in <module>
>     class IPv4Address(object):
>   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/address.py", line 37, in IPv4Address
>     type = attr.ib(validator=attr.validators.in_(["TCP", "UDP"]))
> **AttributeError: 'module' object has no attribute 'in_'**

## Expected results
As it was working and I installed the new requirements it should just work.
### Debugging tracebacks (current results)

Please attach the result of:

pip freeze > requirements_freeze.txt

[requirements_freeze.txt](https://github.com/infobyte/faraday/files/2418296/requirements_freeze.txt)

Provide information on your operating system. Example:

$ cat /etc/lsb-release
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=18.3
DISTRIB_CODENAME=sylvia
DISTRIB_DESCRIPTION="Linux Mint 18.3 Sylvia"

**Comment by montive** [comment]
Hi there. In order to update Faraday's version, follow this guide: 

https://github.com/infobyte/faraday/wiki/Updates

**Comment by daniejstriata** [comment]
Thanks. I followed the steps to install the requirements and get the error when running the server.

**Comment by montive** [comment]
Hi there. Sorry for the late reply. Can you send us the output of the following command:

$ pip freeze | grep Twisted

**Comment by daniejstriata** [comment]
No problem. Here you go:
Twisted==18.7.0

**Comment by montive** [comment]
Hi. 

You need to install Twisted==18.4.0. You can later update it to its last version but for the moment, lets downgrade it.

**Comment by daniejstriata** [comment]
It was something else. I'm not running Twisted==18.9.0 and Faraday is working. I did have to redo my Database and lost all previous workspaces. The upgrade path on the wiki is not clear and the RELEASE states that one must run:
./faraday.py --update
but that gives me:
faraday.py: error: unrecognized arguments: --update

After logging in to Faraday for the first time and creating a new workspace I tried importing reports from Lynis which fails to import anything but also does not list any errors.

What is the best way to start from scratch?
